test-case/check-volume-levels.sh

```diff
@@ -0,0 +1,386 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2020 Intel Corporation. All rights reserved.
+
+set -e
+
+##
+## Case Name: check-volume-levels
+## Preconditions:
+##    topology is nocodec with loopack in PCM0P -> PCM0C
+##    aplay should work
+##    arecord should work
+##    PCM0C capture PGA supports -50 - +30 dB range
+##    PCM0C capture PGA supports mute switch
+## Description:
+##    Set volume and mute switch to various values, measure
+##    volume gain from the actual levels and compare to set gains.
+## Case step:
+##    1. Start aplay
+##    2a. Capture 1st wav file
+##    2b. Capture 2nd wav file
+##    2c. Capture 3rd wav file
+##    3. Measure volume gains
+## Expect result:
+##    command line check with $? without error
+##
+
+# shellcheck source=case-lib/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")/../case-lib/lib.sh"
+
+OPT_OPT_lst['t']='tplg'     OPT_DESC_lst['t']='tplg file, default value is env TPLG: $''TPLG'
+OPT_PARM_lst['t']=1         OPT_VALUE_lst['t']="$TPLG"
+
+func_opt_parse_option "$@"
+
+tplg=${OPT_VALUE_lst['t']}
+
+TPLGREADER=../tools/sof-tplgreader.py
```

Comment:
I missed this, found it when trying to run the script: test scripts must be runnable from anywhere. Fixed with this change:
```diff
@@ -26,8 +26,10 @@ set -e
 ##    command line check with $? without error
 ##
 
+
+TESTDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)
 # shellcheck source=case-lib/lib.sh
-source "$(dirname "${BASH_SOURCE[0]}")/../case-lib/lib.sh"
+source "$TESTDIR/case-lib/lib.sh"
 
 OPT_OPT_lst['t']='tplg'     OPT_DESC_lst['t']='tplg file, default value is env TPLG: $''TPLG'
 OPT_PARM_lst['t']=1         OPT_VALUE_lst['t']="$TPLG"
@@ -36,7 +38,9 @@ func_opt_parse_option "$@"
 
 tplg=${OPT_VALUE_lst['t']}
 
-TPLGREADER=../tools/sof-tplgreader.py
+TPLGREADER="$TESTDIR"/tools/sof-tplgreader.py
+
+
 PCM_ID=0
 CAP_FORMAT="S16_LE"
 CAP_RATE="48000"

```